builds/e2e/longhaul.yaml

```diff
@@ -337,4 +339,66 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.arm64v8)'
-          longHaul.protocolHead: 'mqtt'
\ No newline at end of file
+          longHaul.protocolHead: 'mqtt'
+          
+################################################################################
+  - job: windows_amd64_moby
+################################################################################
+    displayName: Windows AMD64 Moby
+    condition: eq(variables['run.windows.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
```

Comment:
Please help to add new Windows VM to OneNote, https://microsoft.sharepoint.com/teams/Azure_IoT/_layouts/OneNote.aspx?id=%2Fteams%2FAzure_IoT%2FShared%20Documents%2FEngineering%2FIoTHubNotes&wd=target%28Azure%20IoT%20Edge%2FWiki.one%7C70253E27-7EC5-43AD-89E4-C9BA3943D8CF%2FIoT%20Edge%20VMs%5C%2FDevices%7C0330C378-E885-4582-A9ED-C80CBEA3F05F%2F%29
onenote:https://microsoft.sharepoint.com/teams/Azure_IoT/Shared%20Documents/Engineering/IoTHubNotes/Azure%20IoT%20Edge/Wiki.one#IoT%20Edge%20VMs/Devices&section-id={70253E27-7EC5-43AD-89E4-C9BA3943D8CF}&page-id={0330C378-E885-4582-A9ED-C80CBEA3F05F}&end